### ISSUE TITLE
v5.0

### ISSUE THREAD

**Issue by rodchenkov** [FortAwesome/Font-Awesome#12227]
Can you please provide instruction on how to use FA v5.0 from npm?

Thank you!

**Comment by wolfy1339** [comment]
This info is on the website: https://fontawesome.com

**Comment by rodchenkov** [comment]
@wolfy1339 thanks, I know website but there no npm or bower mentioned.
I found nmp packages here: https://www.npmjs.com/~robmadole
But I don't see any docs how to use it and when I try them myself it is not working.

**Comment by wolfy1339** [comment]
Bower support is gone.
You just had to go in the getting started section: https://fontawesome.com/get-started/advanced-options

**Comment by rodchenkov** [comment]
@wolfy1339 thank you.
Just want to ask @robmadole about this https://www.npmjs.com/~robmadole
@robmadole can you help me with your npm packages?

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

#### Installation with official NPM packages
Free version: https://fontawesome.com/how-to-use/use-with-node-js#free
Pro version: https://fontawesome.com/how-to-use/use-with-node-js#pro

Instructions are in the same page

Hope it helps.

Closing here

**Comment by rodchenkov** [comment]
Hi @tagliala 
Thank you! What about NPM package for HTML projects?
I want to use fontawesome NPM package also in dotnet core MVC project.

**Comment by tagliala** [comment]
@rodchenkov you're welcome

> I want to use fontawesome NPM package also in dotnet core MVC project.

Sorry but I can't help. I'm going to reopen here with a proper title but this looks like a support request

**Comment by robmadole** [comment]
We don't have any experience with .NET projects. Perhaps there is someone else in the community that could help you out. We'll leave this issue open.

**Comment by rodchenkov** [comment]
@robmadole @tagliala thank you for your help.
As my MVC project is like usual HTML website, looks like all I need is just NPM or Bower package that will have this file:
/static/fontawesome/fontawesome-all.js
like in your example here:
https://fontawesome.com/get-started/svg-with-js

Is that possible?

**Comment by robmadole** [comment]
Yep, sure. You can download the .ZIP or use the NPM packages. All that is available from https://fontawesome.com

**Comment by rodchenkov** [comment]
@robmadole 
can you tell me witch NPM package contains: /static/fontawesome/fontawesome-all.js?
Thank you

**Comment by robmadole** [comment]
We don’t have a package that contains that file. Not yet anyway.

**Comment by rodchenkov** [comment]
Thank you @robmadole 
Are you planing to make such NPM package?

**Comment by robmadole** [comment]
Yes, soon @rodchenkov

**Comment by nk-o** [comment]
Hi @robmadole,
Moved now from 4 version and can't find fontawesome-all.min.js file in npm package. Really need it in HTML project.

**Comment by robmadole** [comment]
We’ll be addressing this as part of 5.1.0 which is our top priority right now.

**Comment by robmadole** [comment]
Ok, we have a new package `@fortawesome/fontawesome-free` and `@fortawesome/fontawesome-pro` (for Pro customers) that include the minified files that we make available from the CDN and Pro CDN.

@nk-o if you get a chance we'd like to see if this addresses your needs.

**Comment by nk-o** [comment]
@robmadole yep, what I was waiting for. Thank you.
